aws-lambdas/iac/terraform/lambda/main.tf

```diff
@@ -1,3 +1,17 @@
+terraform {
+  required_version = ">= 1.0.8"
```

Comment:
Don't make this a hard constraint. Change to `required_version = "~> 1.3"`

Reply:
done